server/index.js

```diff
@@ -5,13 +5,24 @@ const port = parseInt(process.env.PORT, 10) || 3000;
 const dev = process.env.NODE_ENV !== 'production';
 const app = next({ dev });
 const handle = app.getRequestHandler();
+const multer = require('multer');
+const path = require('path');
 
 const content = require('./content');
 const contentType = require('./content-type');
 
+const storage = multer.diskStorage({
+  destination: './static',
+  filename(req, file, cb) {
+    cb(null, `${file.originalname}`);
+  },
+});
+
+const upload = multer({ storage });
+
 app.prepare().then(() => {
   const server = express();
-
+  server.use(express.static(path.join(__dirname, '..', 'static')));
```

Comment:
I'm not really sure if we need this line. Looks like next.js is taking care of it - https://nextjs.org/docs#static-file-serving-eg-images . What am I missing?